src/plugins/data/server/index_patterns/fetcher/lib/merge_capabilities_with_fields.ts

```diff
@@ -56,16 +56,21 @@ export const mergeCapabilitiesWithFields = (
       rollupFields.push(
         ...fields
           .filter((field) => !rollupFieldNames.includes(field))
-          .map((field) => {
+          .reduce<FieldDescriptor[]>((collector, field) => {
```

Comment:
Swapping `map` for `reduce` since we don't want to return fields missing in the field caps info.